sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/implementation/models/QueueDescriptionImpl.java

```diff
@@ -121,29 +115,26 @@ public final class QueueDescription {
     private Integer messageCount;
 
     /*
-     * A value indicating if the resource can be accessed without
-     * authorization.
+     * A value indicating if the resource can be accessed without authorization.
      */
     @JacksonXmlProperty(
             localName = "IsAnonymousAccessible",
             namespace = "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect")
     private Boolean isAnonymousAccessible;
 
     private static final class AuthorizationRulesWrapper {
-        @JacksonXmlProperty(localName = "AuthorizationRule",
-            namespace = "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect")
-        private final List<AuthorizationRuleImpl> items;
 
-        @JsonCreator
-        private AuthorizationRulesWrapper() {
-            this.items = Collections.emptyList();
-        }
```

Comment:
Looks like the default Ctr patch introduced in the pr https://github.com/Azure/azure-sdk-for-java/pull/29508 got overwritten by the autorest. :(

Reply:
Urgh. Let me add that into the customisations as well.